fix: update docker compose and go-ftw packages install (#167)
coreruleset/documentation: content/development/testing.md

For that you will need:

- the CRS Git repository
- Docker compose. See [here](https://docs.docker.com/compose/install/) for installation instructions.
  OR
  [setup and use your own environment]({{% relref "#use-own-env" %}})
- your rules and tests!

{{% notice style="primary" title="Installing Go-FTW" icon="fa-solid fa-lightbulb" %}}
We strongly suggest to install a pre-compiled binaries of **go-ftw** available [on GitHub](https://github.com/coreruleset/go-ftw/releases).

> The binaries are ready to run and do not require installation. In the releases page you will also find `.deb` and `.rpm` packages that can be used in some Linux OS.

Reviewer: ```suggestion
The binary is ready to run and does not require installation. On the releases page you will also find `.deb` and `.rpm` packages that can be used for installation on some GNU/Linux systems.
```
